addons/mail/static/src/models/channel_command/channel_command.js

```diff
@@ -3,10 +3,104 @@ odoo.define('mail/static/src/models/channel_command/channel_command.js', functio
 
 const { registerNewModel } = require('mail/static/src/model/model_core.js');
 const { attr } = require('mail/static/src/model/model_field.js');
+const { cleanSearchTerm } = require('mail/static/src/utils/utils.js');
 
 function factory(dependencies) {
 
-    class ChannelCommand extends dependencies['mail.model'] {}
+    class ChannelCommand extends dependencies['mail.model'] {
+
+        /**
+         * Fetches channel commands matching the the given search term to extend
```

Comment:
typo repetition _"the the"_

Reply:
Haha I greped the error to find it and I'm far from being the only one in our code.